tcmba/spiders/obras.py

```diff
@@ -0,0 +1,127 @@
+from datetime import date, datetime
+
+from scrapy import Request, Spider
+from scrapy.selector import Selector
+
+from tcmba.items import ConstructionItem
+from tcmba.spiders.helpers import format_year
+
+
+def get_span_value_for_label(response, label):
+    value = (
+        response.css(".form-group")
+        .xpath(f".//span[preceding-sibling::label[contains(., '{label}')]]/text()")
+        .get()
+    )
+    return value if value else ""
```

Comment:
```suggestion
    return value or ""
```
Você pode simplificar aqui.